collectors.rb

```diff
@@ -57,6 +57,34 @@ def collect_ganeti(cluster)
     @node_cache.write
   end
 
+  def collect_chef(url, client, key)
+    chef_template = ERB.new File.new('cache_templates/chef_template.erb').read,
+                            nil, '%'
+
+    ChefAPI::Connection.new(endpoint: url, client: client, key: key) do |connection|
+      connection.nodes.each do |n|
+        node_name = n.name || 'unknown'
```

Comment:
Are we going to have collisions if multiple nodes don't have a name? Do we know if it is even possible for nodes not to have a name?